src/test/kotlin/no/nav/syfo/personoppgave/api/v2/VeilederPersonOppgaveApiV2Spek.kt

```diff
@@ -147,8 +149,8 @@ class VeilederPersonOppgaveApiV2Spek : Spek({
                 }
             }
 
-            describe("Process PersonOppgave for PersonIdent") {
-                it("returns OK and does NOT send Personoppgavehendelse if processed 1 of 2 existing PersonOppgave") {
+            describe("Process OppfolgingsplanLSP-PersonOppgave for PersonIdent") {
```

Comment:
Det er vel LPS her også  (forkortelse for Lønns- og Personalsystem)

Reply:
Ja, bra sett!